tests/test_trnsys.py

```diff
@@ -23,6 +32,48 @@ def step_forward(self, steps: int) -> (bool, int):
             return (False, 1)
         return (True, 0)
 
+    def get_output_value(self, unit: int, output_number: int) -> (float, int):
+        if unit not in self.units:
+            return (math.nan, 1)
+        key = f"{unit}:{output_number}"
+        value = self.unit_outputs.get(key)
+        if value is None:
+            return (math.nan, 2)  # output is not available for this unit
+        return (value, 0)
+
+    def set_input_value(self, unit: int, input_number: int, value: float) -> int:
```

Comment:
Shouldn't this always return an int? I don't think it actually does. Should it return 0 if successful?

Reply:
It should return a 0 on success.  Fixed in fa8cc1a